en/docs/governance/rule-validator/rule-validator.md

```diff
@@ -0,0 +1,588 @@
+# WSO2 Rule Validator
+
+---
+- WSO2 Rule Validator is a JSON/YAML linter that is a Java implementation of Stoplight Spectral. [https://github.com/stoplightio/spectral](https://github.com/stoplightio/spectral)
+- This validator is automatically called by the APIM Governance feature to validate different artifacts. (Open API Specifications, API Metadata files, etc.)
+- Any valid JSON or YAML file can be validated by providing a valid Spectral ruleset.
+
+## Features and Limitations
+WSO2 Rule validator supports most features that Spectral itself does. With the exception of a few listed below.
+
+1. [_**given**_](https://docs.stoplight.io/docs/spectral/d3482ff0ccae9-rules#rules-properties) path is a JSON Path. But currently WSO2 Rule Validator does not support [JSON Path Plus](https://github.com/JSONPath-Plus/JSONPath) features, even though Spectral does.
```

Comment:
Validator 'V' should be capital.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/9249